Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM gcr.io/metamarkets-prod-xpn-host/node:8.9.4-alpine
```

Comment:
if it's an easy update, we should try to run this on node 10.16.0, the current LTS version.  I'm planning to get all application services on that node version.

Reply:
furthermore, node:8.9.4-alpine doesn't exist in our container registry, and even if it did, the alpine images don't have python and some other things needed to do the `npm install`.

The current version of the base image I've been building for application services is `gcr.io/metamarkets-prod-xpn-host/node:10.16.1-base-1`.  That's `10.16.1-alpine` with some additional stuff like python installed (can't remember exactly what, have to grab the Dockerfiles off my old laptop).  docker-console appears to build correctly using that image.